code/game/jobs/job/event.dm

```diff
@@ -0,0 +1,121 @@
+//Generic department roles for events where organization may be required.
+/datum/job/eventsec
```

Comment:
Why are they not just subtypes of the already existing departments? It looks like there's a wealth of copy-pastes from the normal roles - why not make it a subtype?

Reply:
Having not really touched job code before, I noticed that subtypes of jobs doesn't seem to be a thing that's really done anywhere outside of the consular/liaison.

Reply:
Yes, this ancient relic of code seems to all be structured this way, we will maybe one day refactor it, for now I think this is fine enough